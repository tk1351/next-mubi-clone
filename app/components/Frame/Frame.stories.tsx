import type { Meta, StoryObj } from "@storybook/react";
import Image from "next/image";
import { Frame } from "./Frame";

const meta = {
  title: "Components/Frame",
  component: Frame,
} satisfies Meta<typeof Frame>;

export default meta;
type Story = StoryObj<typeof meta>;
export const Primary: Story = {
  args: {
    children: (
      <Image
        src="https://picsum.photos/200/300"
        alt=""
        width={300}
        height={300}
      />
    ),
  },
};
