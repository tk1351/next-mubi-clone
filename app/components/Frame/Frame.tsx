import { type FC, type PropsWithChildren } from "react";
import styles from "./Frame.module.css";

export type Props = PropsWithChildren<{}>;

export const Frame: FC<Props> = ({ children }) => {
  return <div className={styles.frame}>{children}</div>;
};
