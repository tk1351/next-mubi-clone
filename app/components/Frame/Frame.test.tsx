import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";
import { Frame } from "./Frame";

describe(`Testing Frame Component`, () => {
  it("Snapshot", () => {
    const { container } = render(<Frame />);
    expect(container).toMatchSnapshot();
  });
});
