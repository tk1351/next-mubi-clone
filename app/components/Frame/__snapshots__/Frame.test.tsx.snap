// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Testing Frame Component > Snapshot 1`] = `
<div>
  <div
    class="_frame_207b3f"
  />
</div>
`;
